modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1575,7 +1583,7 @@ public String toApiName(String name) {
         if (name.length() == 0) {
             return "DefaultApi";
         }
-        return camelize(name) + "Api";
+        return (this.apiSuffix.isEmpty() ? camelize(name) : camelize(name) + this.apiSuffix);
```

Comment:
that would be great to remove the `toApiName` overriding method in `AbstractJavaCodegen` class to be able to have this feature in most of java generators